lib/board.rb

```diff
@@ -0,0 +1,22 @@
+class Board
+  def initialize
+    @tiles = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+  end
+
+  def show
```

Comment:
This is a UI related method and may not be the responsibility anymore of the Board class